files/zh-cn/web/api/element/copy_event/index.md

```diff
@@ -1,52 +1,74 @@
 ---
-title: copy
+title: Element：copy 事件
+short-title: copy
 slug: Web/API/Element/copy_event
 ---
 
-当用户通过浏览器 UI（例如，使用 <kbd>Ctrl</kbd>/<kbd>⌘</kbd>+<kbd>C</kbd> 键盘快捷方式或从菜单中选择“复制”）启动复制操作并响应允许的{{domxref("Document.execCommand","document.execCommand('copy')")}}调用时触发`copy`事件。
+{{APIRef}}
 
-## 基本信息
+当用户通过浏览器的用户界面发起复制动作时，将触发 **`copy`** 事件。
 
-- Specification
-  - : [Clipboard](https://www.w3.org/TR/clipboard-apis/#the-copy-action)
-- Interface
-  - : {{domxref("ClipboardEvent")}}
-- Bubbles
-  - : Yes
-- Cancelable
-  - : Yes
-- Target
-  - : {{domxref("Element")}}：获得焦点的元素（如{{domxref("HTMLElement.contentEditable","contentEditable")}}内容能编辑或者可以选中的元素），或{{HTMLElement("body")}}。
-- Default Action
-  - : 见下文。
+该事件的默认行为是将当前选中的内容（如有）复制到系统剪贴板。
 
-调用{{domxref("DataTransfer.setData","setData(format, data)")}}可以修改{{domxref("ClipboardEvent.clipboardData")}}事件的默认行为：
+事件处理器可以通过调用 {{domxref("ClipboardEvent.clipboardData")}} 属性上的 {{domxref("DataTransfer.setData", "setData(format, data)")}} *修改*剪贴板内容，并通过 {{domxref("Event/preventDefault", "event.preventDefault()")}} 取消默认行为。
+
+然而，事件处理器不可以*读取*剪贴板数据。
+
+可以构建和调度一个[合成](/zh-CN/docs/Web/Events/Creating_and_triggering_events) `copy` 事件，但这不会影响系统剪贴板。
```

Comment:
https://developer.mozilla.org/zh-CN/docs/Web/API/EventTarget/dispatchEvent

Same in the rest of documents, other wise looks good 👍

```suggestion
可以构建和派发一个[合成](/zh-CN/docs/Web/Events/Creating_and_triggering_events) `copy` 事件，但这不会影响系统剪贴板。
```